package.json

```diff
@@ -120,6 +120,8 @@
     "eslint-plugin-react": "^7.0.1",
     "extract-text-webpack-plugin": "^3.0.2",
     "ignore-styles": "^5.0.1",
+    "node-sass": "^4.7.2",
+    "sass-loader": "^6.0.6",
```

Comment:
Lets remove `react-css-modules` from package.json too!

Reply:
I'll add that to the todos; I want to make sure that frontend works before I start breaking backend haha

Reply:
Also, in order to make `jest` play nice with the `import ... .scss` files during unit testing we will have to change this line:

```
"\\.(css|less)$": "identity-obj-proxy",
```
to this
```
"\\.(css|less|scss)$": "identity-obj-proxy",
```
below in packege.json